src/ocf_io.c

```diff
@@ -86,6 +86,10 @@ struct ocf_io *ocf_io_new(ocf_volume_t volume, ocf_queue_t queue,
 		uint32_t io_class, uint64_t flags)
 {
 	struct ocf_io_internal *ioi;
+	uint32_t sector_size = 1 << ENV_SECTOR_SHIFT;
```

Comment:
We have dedicated macros for converting between sectors and bytes:
`uint32_t sector_size = SECTORS_TO_BYTES(1);`